scipy/linalg/lapack.py

```diff
@@ -722,6 +765,30 @@
 }
 
 
+# Place guards against markdown problems with special characters
+p1 = regex_compile(r'with bounds (?P<b>.*?)( and (?P<s>.*?) storage){0,1}\n')
+p2 = regex_compile(r'Default: (?P<d>.*?)\n')
+
+
+def backtickrepl(m):
+    if m.group('s'):
+        return ('with bounds ``{}`` with ``{}`` storage\n'
+                ''.format(m.group('b'), m.group('s')))
+    else:
+        return 'with bounds ``{}``\n'.format(m.group('b'))
+
```

Comment:
```
scipy/linalg/lapack.py:780:1: E305 expected 2 blank lines after class or function definition, found 1
1       E305 expected 2 blank lines after class or function definition, found 1
```
This is the Travis failure.

Reply:
> If everything goes OK, after the reviews I'd like to go through `test_decomp.py` for PEP8 violations.

I'm not sure if there's actually a problem with `test_decomp.py`, although I agree the TravisCI message is confusing.

Reply:
There is another error about unicodeness of lambda char after this error. 

`test_comp.py` is a separate issue. It really needs some PEP8 love.